Gems/AtomLyIntegration/CommonFeatures/Code/Include/AtomLyIntegration/CommonFeatures/Material/MaterialComponentBus.h

```diff
@@ -19,7 +19,7 @@ namespace AZ
         {
         public:
             //! Get a map representing the default layout and values for all material assignment slots on the source model or object
-            virtual MaterialAssignmentMap GetDefautMaterialMap() const = 0;
+            virtual MaterialAssignmentMap GetDefaultMaterialMap() const = 0;
```

Comment:
I hate to say this, but you should probably add a deprecated function with the old spelling that by default calls the new one. Otherwise, any code out there relying on the old spelling will immediately break.